### DISCUSSION TITLE
Change Date/Time Format  vs. Templating in Home Assisant

### DISCUSSION THREAD

**Discussion by AdmiralRaccoon** [netalertx/NetAlertX#714] [Q&A]
Hello,
I´m not sure if that might be a linux limitation (or maybe it even has something to do with the other Post ["Time+8"](https://github.com/jokob-sk/NetAlertX/discussions/695)

This is the Sensor ouput NetAlertX sends to Home Assistant
`nokiag21-carport last_connection : 
2024-06-26 20:20:29+02:00`

Now, I created a Template sensor to get this output:
`Nokia G21 Carport Last Seen : 26.06. 20:20`

while this is the template code:
`{{ as_timestamp(states('sensor.nokiag21_carport_last_connection')) | timestamp_custom('%d.%m. %H:%M') }}`

1.) Is it possible for NetAlertX to send a custom mqtt  timestamp value itself, so there would be no need for templating in Home Assistant?

2.) If not, and since I´m not a templating god, neither have the ambitions to create so many template sensors manually, is there a known way in Home Assistant to bulk change / bulk template multiple entities at once?

Thanks for Ideas,
Philipp

**Comment by jokob-sk** [comment]
Hey,

I implemented a change to remove the +TZ from the dev_LastConnection column. This will be gradually updated once advice is detected as offline and comes back online.

This should be available in the next release. Please have a look at the `netalertx-dev` docker image, in about 15 minutes (or after the last [action finishes](https://github.com/jokob-sk/NetAlertX/actions)) from now. 

It would be great if you could test this (backup everything first or use a new container) on your end by switching to the above image and letting me know if the issue was resolved.

Thanks in advance,
j

**Reply by AdmiralRaccoon** [reply]
hey, I´ve tested the dev image, but the container is restarting in loop with this error:

```
`16:30:16 Permissions check (All should be True)
16:30:16 ------------------------------------------------
16:30:16   /config/app.conf |  READ  | True
16:30:16   /config/app.conf |  WRITE | True
16:30:16   /db/app.db       |  READ  | True
16:30:16   /db/app.db       |  WRITE | True
16:30:16 ------------------------------------------------
16:30:16 [Setup] Attempting to fix permissions.
16:30:16 [Setup] Attempting to fix permissions.
16:30:16 [Database] Opening DB
16:30:16 [upgradeDB] Re-creating Settings table
16:30:16 [upgradeDB] Re-creating Pholus_Scan table
16:30:16 [upgradeDB] Re-creating Parameters table
16:30:16 [Config] reading config file
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/app/server/__main__.py", line 209, in <module>
    sys.exit(main())       
             ^^^^^^
  File "/app/server/__main__.py", line 92, in main
    all_plugins = importConfigs(db, all_plugins)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/server/initialise.py", line 125, in importConfigs
    c_d = read_config_file(config_file)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/server/initialise.py", line 332, in read_config_file
    code = compile(filename.read_text(), filename.name, "exec")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "app.conf", line 48
    SCAN_SUBNETS__metadata=
                           ^
SyntaxError: invalid syntax`
```

**Reply by jokob-sk** [reply]
Hi,
could you please double check your conf file is in the right format? From the error it looks to be corrupted.

**Reply by AdmiralRaccoon** [reply]
Looks good to me:

![image](https://github.com/jokob-sk/NetAlertX/assets/41727850/d37a1ccd-c592-41e7-a5cd-e5b7fb6521f5)

**Reply by jokob-sk** [reply]
Great, thanks for checking!
